book/src/proposals/tick_verification.md

```diff
@@ -0,0 +1,65 @@
+# Tick Verification
+
+This design the criteria and validation of ticks in a slot. It also describes
+error handling and slashing conditions encompassing how the system handles
+transmissions that do not meet these requirements.
+
+# Slot structure
+
+Each slot must contain an expected `ticks_per_slot` number of ticks. The last
+shred in a slot must contain only the entirety of the last tick, and nothing 
+else. The leader must also mark this shred containing the last tick with the 
+`LAST_SHRED_IN_SLOT` flag. Between ticks, there must be `hashes_per_tick`
+number of hashes.
+
+# Handling bad transmissions
+
+Malicious transmissions `T` are handled in two ways:
+
+1) If a leader can generate some erronenous transmission `T` and also some 
+alternate transmission `T'` for the same slot without violating any slashing
+rules for duplicate transmissions (for instance if `T'` is a subset of `T`),
+then the cluster must handle the possibility of both transmissions being live.
+
+Thus this means we cannot mark the erronenous transmission `T` as dead because
+the cluster may have reached consensus on `T`. These cases necessitate a 
+slashing proof to punish this bad behavior.
+
+2) Otherwise, we can simply mark the slot as dead and not playable. A slashing
+proof may or may not be necessary depending on feasibility.
+
+# Blocktree receiving shreds
+
+When blocktree receives a new shred `s`, there are two cases:
+
+1) `s` is marked as `LAST_SHRED_IN_SLOT`, then check if there exists a shred 
```

Comment:
Is it possible for the blocktree to receive two different shreds for the same index?

Reply:
Seems like we should add a case for that too

Reply:
Nvm I see we have a separate proposal for this case (duplicate blocks) maybe link to it?

Reply:
ya that one's a nasty bugger. It hasn't been checked in yet, you mean link it here in this PR?

Reply:
Oh I meant link in the book to the other proposal so that all the cases are covered for how to handle a new shred. Or for now, say proposal pending?